Makefile.envs

```diff
@@ -17,6 +17,3 @@ export PYTHONINCLUDE=$(PYODIDE_ROOT)/cpython/installs/python-$(PYVERSION)/includ
 
 # Use env variable if defined, otherwise fallback to './'
 export PYODIDE_BASE_URL?=./
-# This env variable is used to detect pyodide at build time,
-# do not rename it.
-export PYODIDE_PACKAGE_ABI=1
```

Comment:
(My comment didn't seem to be saved, so re-writing it).

The issue with removing this env variable, is that we have contributed fixes with it to upstream packages (at least scikit-image and scikit-learn) as a way to detect building it pyodide. Maybe we could keep it here for now, with a comment and open a separate issue on how to properly detect building in Pyodide (also taking into account that we might use cross-compilation in the future). Using `PYODIDE_BASE_URL` would be OK for this PR, but I'm not sure it's a good solution long term.

Reply:
We could introduce a `PYODIDE` environment variable specifically for this purpose